src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -64,57 +64,6 @@ function transformApplicants(applicants) {
   return applicantsPostTransform;
 }
 
-// Since the certifier data may be the sponsor's or a third party, this maps
-// the sponsor's info into the certifier property names for simplicity on BE
-function parseCertifier(transformedData) {
-  return {
-    date: new Date().toJSON().slice(0, 10),
-    firstName: transformedData.veteransFullName.first || '',
-    lastName: transformedData.veteransFullName.last || '',
-    middleInitial: transformedData?.veteransFullName?.middle || '',
-    phoneNumber: transformedData?.sponsorPhone || '',
-    relationship: 'sponsor',
-    streetAddress: transformedData?.sponsorAddress?.streetCombined || '',
-    city: transformedData?.sponsorAddress?.city || '',
-    state: transformedData?.sponsorAddress?.state || '',
-    postalCode: transformedData?.sponsorAddress?.postalCode || '',
-  };
-}
-
-// Set up the `primaryContactInfo` for the backend notification API service.
-function getPrimaryContact(data) {
-  // If a certification name is present, third party signer is the certifier
-  const useCert =
-    data?.certification?.firstName && data?.certification?.firstName !== '';
-
-  // Either the first applicant with an email address, or the first applicant
-  const primaryApp =
-    data?.applicants?.filter(
-      app => app.applicantEmailAddress && app.applicantEmailAddress.length > 0,
-    )[0] ?? data?.applicants?.[0];
-
-  return {
-    name: {
-      first:
-        (useCert
-          ? data?.certification?.firstName
-          : primaryApp?.applicantName?.first) ?? false,
-      last:
-        (useCert
-          ? data?.certification?.lastName
-          : primaryApp?.applicantName?.last) ?? false,
-    },
-    email:
-      (useCert
-        ? data?.certification?.email
-        : primaryApp?.applicantEmailAddress) ?? false,
-    phone:
-      (useCert
-        ? data?.certification?.phoneNumber
-        : primaryApp?.applicantPhone) ?? false,
-  };
-}
-
```

Comment:
Able to remove this logic since we no longer have any doubt about what info belongs to the form signer/certifier. Additionally, since we're requiring an email address up front from whoever is filling the form all the `getPrimaryContact` logic can go away.